ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -37,6 +42,13 @@ ethernet_interfaces:
       unknown_unicast:
         level: 1
         unit: percent
+    priority_flow_control: 
+      enabled: true
+      priorities: 
+        - priority: 5
+          no_drop: true # no-drop
+
+    
```

Comment:
Please remove two blank lines 50,51. Just one blank like (already there, line 52) is recommended between two different blocks